dakota_manylinux_install_files/CMakeLists.txt

```diff
@@ -0,0 +1,316 @@
+dakota_find_eigen3()
+# Conditionally find GSL; needed by both packages/ (QUESO) and src/
+dakota_find_gsl()
+# Conditionally find HDF5; needed by both src/ and test/
+dakota_find_hdf5()
+# Unconditionally find Boost
+dakota_find_boost()
+
+
+include(CTest)
+
+
+# Set the export name for install targets; parent packages likely want
+# to override this to the name of their target
+set(ExportTarget DakotaTargets CACHE STRING 
+  "Export set name for ${CMAKE_PROJECT_NAME}")
+
+# Unconditionally find Trilinos/Teuchos, optionally ROL
+include(DakotaFindTrilinos)
+dakota_find_trilinos()
+
+# Conditionally find X-Windows graphics deps
+include(DakotaXGraphics)
+dakota_find_x_graphics()
+
+if(DAKOTA_F90)
+  FortranCInterface_HEADER(dak_f90_config.h MACRO_NAMESPACE DAK_F90_)
+  install(FILES ${Dakota_BINARY_DIR}/dak_f90_config.h DESTINATION include)
+endif(DAKOTA_F90)
+
+# Initialize GCov before adding targets to cover
+include(DakotaGcov)
+dakota_gcov_init()
+
+# -----
+# Configuration of Dakota components
+# -----
+
+# --- New surrogates module
+# BMA TODO: Finer granularity in packages to build Trilinos, Eigen
+# first, then optionally Pecos, Surfpack, surrogates.
+if(DAKOTA_MODULE_UTIL)
+  add_subdirectory(src/util)
+endif()
+
+if(DAKOTA_MODULE_SURROGATES)
+  add_subdirectory(src/surrogates)
+  add_definitions("-DHAVE_DAKOTA_SURROGATES")
+endif()
+
+if(DAKOTA_MODULE_DAKOTA)
+
+  add_subdirectory(packages)
+
+  add_subdirectory(src)
+  # Build the executables in the test directory even if BUILD_TESTING is OFF
+  if(DAKOTA_ENABLE_TESTS)
+    add_subdirectory(test)
+  endif()
+
+  if(ENABLE_DAKOTA_DOCS)
+    add_subdirectory(docs)
+  endif()
+
+  # Legacy examples
+  add_subdirectory(examples)
+  # New examples library, taking care to guard against empty directory
+  if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/dakota-examples/CMakeLists.txt")
+    add_subdirectory(dakota-examples)
+  endif()
+
+  add_subdirectory(interfaces)
+
+  # Install helper scripts to bin/:
+  # NOTE: This directory manages dynamic library manipulation at install time,
+  # as the install script must be called after src/ and test/ are installed.
+  add_subdirectory("scripts")
+
+endif()  # DAKOTA_MODULE_DAKOTA
+
+
+# -----
+# Install / Package
+# -----
+
+# Top-level install rules from source (binary rules are in subdirs)
+install(FILES INSTALL LICENSE COPYRIGHT README
+  DESTINATION ${DAKOTA_TOPFILES_INSTALL}
+  )
+
+install(DIRECTORY examples DESTINATION ${DAKOTA_EXAMPLES_INSTALL}
+  USE_SOURCE_PERMISSIONS
+  PATTERN "CMakeLists.txt" EXCLUDE
+  PATTERN "*.c" EXCLUDE 
+  PATTERN "*.f" EXCLUDE 
+  )
+
+install(DIRECTORY test DESTINATION ${DAKOTA_TEST_INSTALL}
+  PATTERN "CMakeLists.txt" EXCLUDE
+  PATTERN "*.c" EXCLUDE 
+  PATTERN "*.cpp" EXCLUDE 
+  PATTERN "*.f" EXCLUDE 
+  )
+
+if(DAKOTA_VERSION_FOUND)
+   install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/VERSION DESTINATION
+    ${DAKOTA_TOPFILES_INSTALL})
+endif()
+ 
+# Final CPack settings
+dakota_cpack_finalize()
+include(CPack)
```

Comment:
Please remove this file.
It was unintentionally re-added in commit 4dfd3cc6ef70fd2f0f9e54e4484a6fab90a634c5

Reply:
Removed